.travis.yml

```diff
@@ -1,5 +1,7 @@
 language: cpp
-sudo: false
+sudo: required
+dist: trusty
+group: edge
 os:
   - linux
   - osx
```

Comment:
I suspect something is wrong with the osx_image thing, because the osx binaries from travis crash